.github/CODEOWNERS

```diff
@@ -1,3 +1,25 @@
 CHANGES.md @wmorgan @klingerf @olix0r @admc
 GOVERNANCE.md @klingerf @olix0r
 MAINTAINERS.md @klingerf @olix0r
+
+/.github/workflows/* @alpeb @kleimkuhler
```

Comment:
Just curious, does this path need the wildcard? It doesn't look like we're including it for any of the other paths.

Reply:
Not AFAIK, good call!